index.html

```diff
@@ -7,17 +7,23 @@
   <div class="wrapper">
 
     <div class="welcome">
+      <!-- Lightweight, sound static typing for the life of your code. -->
+      <h1>Confidently write code for JavaScript environments</h1>
```

Comment:
Most of the other feedback I've received is that the wording on the front page needs to be shorter and crisper.  Given that, I think this is heavy.  I do like putting the main value prop above the warning though.  Hm.